File: javascript/q13.js
// Task: Fix the function to print a centered pyramid pattern with correct spacing.

function printPyramid(rows) {
    for (let i = 0; i < rows; i++) {
        let line = "";
        for (let j = 0; j < i; j++) { 
            line += " ";
        }
        for (let k = 0; k < 2 * i + 1; k++) {
            line += "*";
        }
        console.log(line);
    }
}
/*
Test Cases:
printPyramid(5);
Expected Output:
    *
   ***
  *****
 *******
*********
*/

File: python/q13.py
# Task: Fix the function to print a centered pyramid pattern with correct spacing.

def print_pyramid(rows):
    for i in range(rows):
        line = ' ' * i 
        line += '*' * (2 * i + 1)
        print(line)

print_pyramid(3)
"""
Test cases 1:
print_pyramid(3)
Output:
    *
   ***
  *****
"""